samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,186 @@
+---
+page_type: sample
+description: This sample showcases how to utilize Azure Open AI Search within Teams Toolkit for Visual Studio Code to extract Action Items and Meeting summaries from meeting transcriptions.
+products:
+- office-teams
+- copilot-m365
+languages:
+- javascript
+---
```

Comment:
This portion is missing here:

extensions:
contentType: samples
createdDate: "07/15/2024 01:38:25 PM"
urlFragment: officedev-microsoft-teams-samples-bot-ai-virtual-assistant-nodejs

Refer this PR: https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/1335/files#diff-820624b3d7f7e4be48d0fc48df9ab543ee07089829a4bf35bd2e808c12637bb9

Reply:
Done